cms/templates/import.html

```diff
@@ -102,8 +102,8 @@ <h3 class="title">
           <ol class="status-progress list-progress">
             <li class="item-progresspoint item-progresspoint-upload is-complete">
               <span class="deco status-visual">
-                <i class="icon fa fa-cog"></i>
-                <i class="icon fa fa-warning"></i>
+                <span class="icon fa fa-cog" aria-hidden="true"></span>
```

Comment:
Oh, is the text added dynamically? I didn't see it in the HTML, and thus thought there was no text. If text is present, I agree that no sr text is needed.